R/newvalues.R

```diff
@@ -143,6 +143,9 @@ print.check_new_values <-
     invisible(x)
   }
 
+#' @rdname tidy.recipe
+#' @param x A `check_new_values` object.
```

Comment:
I don't think this `@param` tag is required

Unfortunately I have discovered that we have used this param tag in all of the tidy methods, but since they are all documented on the same page, only the _last_ one that roxygen2 sees is actually used as the parameter documentation for `x`, which I think is the one for `step_zv()` since that is last alphabetically.

I think we can really just remove them all (in a separate PR) if we wanted to, and only keep a single source of documentation in `tidy.recipe`. Nevertheless, I'd remove it from here

Reply:
sounds good :+1: